packages/wasm-terminal/lib/wasm-terminal.ts

```diff
@@ -35,11 +32,13 @@ export default class WasmTerminal {
     this.wasmTerminalConfig = new WasmTerminalConfig(config);
 
     // Create our xterm element
-    this.xterm = new Terminal();
+    this.xterm = new Terminal({
+      // rendererType: 'dom'
+    });
     // tslint:disable-next-line
-    this.pasteEvent = this.xterm.on("paste", this.onPaste);
+    // this.pasteEvent = this.xterm.on("paste", this.onPaste);
```

Comment:
Is this handled by xterm now? 🤔

Reply:
Yes, I couldn't see any paste methods there. And copy/paste still worked locally :)